{"name":"test01","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"did not expect to find [0] but found [null]","trace":"java.lang.AssertionError: did not expect to find [0] but found [null]\r\n\tat org.testng.Assert.fail(Assert.java:97)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:969)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:563)\r\n\tat com.lrc.common.BaseTest.assertResponse(BaseTest.java:113)\r\n\tat com.lrc.testcases.Test02.test01(Test02.java:36)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.util.ArrayList.forEach(ArrayList.java:1259)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:770)\r\n\tat org.testng.TestRunner.run(TestRunner.java:591)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:402)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:304)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1102)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1032)\r\n\tat org.testng.TestNG.run(TestNG.java:1000)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"接口请求响应日志","source":"fd952765-3379-405a-8af4-8895decd3e39-attachment"}],"parameters":[{"name":"arg0","value":"CaseInfo{caseId=3, interfaceName='新闻列表', title='传key和page,提取author_name', requestHeader='{\"swagger\":\"1333\",\"Content-Type\":\"application/json\"}', method='post', url='/test/index?key=${key}', inputParams='{\"message2\":${message}}', expected='{\n    \"code\": 0,\n    \"message\": \"修改成功\",\n    \"data\": null,\n    \"extInfo\": null\n}', dbAssert='null', extractExper='{\"author_name\":\"result.data[1].author_name\"}'}"}],"start":1680793058988,"stop":1680793059684,"uuid":"918d28c4-e918-4f93-816e-a00720a6ed40","historyId":"7418b1fdf4e7ebd98f6f8816af2fbd1f","fullName":"com.lrc.testcases.Test02.test01","labels":[{"name":"package","value":"com.lrc.testcases.Test02"},{"name":"testClass","value":"com.lrc.testcases.Test02"},{"name":"testMethod","value":"test01"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"api_auto_test_02"},{"name":"subSuite","value":"com.lrc.testcases.Test02"},{"name":"host","value":"MSI"},{"name":"thread","value":"23636@MSI.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}